sphinx/builders/epub3.py

```diff
@@ -157,30 +169,23 @@ def build_navlist(self, nodes):
                 continue
             if node['level'] > self.config.epub_tocdepth:
                 continue
-            if node['level'] == level:
-                navlist.append(self.new_navlist(node, level))
-            elif node['level'] == level + 1:
-                navstack.append(navlist)
-                navlist = []
-                level += 1
-                if lastnode and self.config.epub_tocdup:
```

Comment:
@shimizukawa How about replacing the default EPUB builder by epub3 on Sphinx-1.5? If agreed, the removal means the `epub_tocdup` option will be deprecated in next major release. Are you okay?